eng/Packaging.props

```diff
@@ -23,12 +23,13 @@
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">4.6.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">4.6.1</PackageVersion>
```

Comment:
This should be `4.7.0`.  We bump minor with releases, 3rd part is for servicing.